sippy-ng/src/App.js

```diff
@@ -383,6 +384,11 @@ export default function App(props) {
                     )}
                   />
 
+                  <Route
+                    path="/componentreadiness/report"
```

Comment:
I'd delete report too.  I think `/componentreadiness` should go to the root, and `/componentreadiness/<component_name>` should go to a page just for that component.  You can't disambiguate the URL if the root page is also /something.